tests/platforms/test_host_collector.py

```diff
@@ -45,6 +47,8 @@ def tearDown(self):
 
         set_agent(self.original_agent)
         set_tracer(self.original_tracer)
+        if '/tmp/.instana/python' in sys.path:
```

Comment:
As mentioned before, if we have a global configuration variable for this path/file name, it will be easier to maintain the code in the future.

Reply:
The hardcoding will be removed.